src/OKRConstants.ts

```diff
@@ -0,0 +1,5 @@
+export class NavigationConstants {
+    public static AreaView = "AreaView";
+    public static DetailView = "DetailView"; 
+    public static Data = "DetailView"; 
```

Comment:
is `data` this intended to have the same value as DetailView?

Reply:
And you made the comment too! Sorry Edwin for saying Mike did it!